setting_up_the_database.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+engine = create_engine('sqlite:///films_db', echo=False)
+
+Base = declarative_base()
+
+class Films(Base):
+    __tablename__ = 'films'
+    id = Column(Integer, primary_key=True)
+    title = Column(String)
```

Comment:
Critical issue: The columns `title`, `director`, and `release_year` currently allow `NULL` values. According to the assignment requirements, these fields are mandatory, so each column must explicitly specify `nullable=False`. Leaving them nullable breaks data integrity and violates the expected database schema.